TFNetworkLayer.py

```diff
@@ -1035,6 +1036,7 @@ def batch_norm(self, data,
     :param tf.Tensor sample_variance:
     :param tf.Tensor gamma:
     :param tf.Tensor beta:
+    :param bool use_frequency: whether to use frequency dim to collect statistics
```

Comment:
what do you mean by "frequency dim"?

Reply:
frequency bin dimension, it's "s:1"

Reply:
No, "s:1" is "s:1". Maybe you have some case where you call this frequency dim for some reason. But it is just "s:1". I don't understand why you think that "s:1" is in general always called a "frequency dim".

Reply:
Then how should I get the frequency bin dim?

Reply:
Well, you could call the parameter `use_spatial1` or so, which is already much better than `use_frequency`.

But this is still not good. This is too specific. What if you want to have `use_spatial2` or so later? You should come up with a generic solution. See e.g. `dropout_noise_shape` as an example. Or the general way to specify axes, `Data.get_axes_from_description`.